src/client/runner/automation/playback/click/index.js

```diff
@@ -0,0 +1,273 @@
+
+                // NOTE: in IE focus is not raised if element was focused
+                // before click, even if focus is lost during mousedown
+                this.eventState.simulateFocus = !browserUtils.isIE || activeElement !== this.eventArgs.element ||
+                                                isContentEditableBody;
+
+                // NOTE: in WebKit and IE mousedown event raising opens select element's dropdown,
+                // therefore we should prevent mousedown and hide the dropdown (B236416)
+                var needCloseSelectDropDown = (browserUtils.isWebKit || browserUtils.isIE) &&
+                                              domUtils.isSelectElement(this.element);
+
+                if (needCloseSelectDropDown)
+                    this._bindMousedownHandler();
+
+                this._bindBlurHandler(activeElement);
+
+                this.eventState.simulateDefaultBehavior =
+                    eventSimulator.mousedown(this.eventArgs.element, this.eventArgs.options);
+
+                if (this.eventState.simulateDefaultBehavior === false) {
+                    this.eventState.simulateDefaultBehavior = needCloseSelectDropDown &&
+                                                              !this.eventState.mousedownPrevented;
+                }
+
+                return this._ensureActiveElementBlur(activeElement);
+            })
+            .then(() => this._focus());
+    }
+
+    _ensureActiveElementBlur (element) {
+        // NOTE: In some cases mousedown may lead to active element changing (browsers raise blur). We simulate
+        // blur event if active element was changed after mousedown and blur event was not raised automatically
+        // (B239273, B253520)
+        return new Promise(resolve => {
+            var simulateBlur = domUtils.getActiveElement() !== element && !this.eventState.blurRaised;
+
+            if (!simulateBlur) {
+                resolve();
+                return;
+            }
+
+            if (browserUtils.isIE && browserUtils.version < 12) {
+                // NOTE: All ways to blur element from client script raise blur
+                // event asynchronously in IE (in MSEdge focus/blur is sync)
+                nextTick()
+                    .then(() => {
+                        if (!this.eventState.blurRaised)
+                            eventSimulator.blur(element);
+
+                        resolve();
+                    });
+            }
+            else {
+                eventSimulator.blur(element);
+                resolve();
+            }
+        });
+    }
+
+    _focus () {
+        if (this.eventState.simulateDefaultBehavior === false)
+            return delay(automationSettings.ACTION_STEP_DELAY);
+
+        // NOTE: For contentEditable elements we should call focus directly for action's element because
+        // option 'caretPos' is indicated for this element and eventArgs.element may be a child of this element
```

Comment:
```
        // NOTE: For contentEditable elements, we should call focus directly for action's element because
        // the 'caretPos' option is indicated for this element and eventArgs.element may be a child of this element
```